docs/DafnyRef/Attributes.md

```diff
@@ -1,14 +1,8 @@
 # 22. Attributes
```

Comment:
Top-level comment: I appreciate you improving the markdown here and filling out some critical information, but shouldn't we be pushing some of this content into the actual output of `/attrHelp`, both in Boogie and in Dafny?

Reply:
I like that idea (that is, making `/attrHelp` give a URL that points into this manual).

Reply:
I implemented that idea (just add a permanent link in /attrHelp). I realized that there are a lot of duplicates, so I reimported the documentation about {:extern} from DafnyOptions because it's much better than what I wrote, and I also added the documentation about `{:print}`

Reply:
Down the road and for a later time, I will probably import others `{:warnShadowing}`, but I really think the documentation should be in one place.